src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -234,6 +232,8 @@ def load_arguments(self, _):
             c.argument('deployment_source_branch', options_list=['--deployment-source-branch', '-b'],
                        help='the branch to deploy')
             c.argument('tags', arg_type=tags_type)
+            c.argument('https_only', help="Redirect all traffic made to an app using HTTP to HTTPS.",
+                       arg_type=get_three_state_flag())
```

Comment:
should the return_label be TRUE here - since this change will impact webapp up where we default this to True?

Reply:
I've tested this and `return_label=True` causes broken behavior when the user calls this as `--https-only false`. Having `return_label=True` results in the value being passed as a string (ie `"true"` or `"false"`) instead of a boolean value (`True` or `False`). This behavior is explained in the documentation in the screenshot below. The result is that if we had `return_label=True` and the user called the command with `--https-only false`, they would have a webapp with httpsOnly true. 

I just doubled check this and it doesn't impact webapps created with `az webapp up` -- they still are httpsOnly by default: 
<img width="959" alt="Screen Shot 2022-07-25 at 12 48 11 PM" src="https://user-images.githubusercontent.com/10121214/180862155-9684ac66-02cf-4ce0-9b94-862ae8108cfb.png">



Documentation for the return_label argument: 
<img width="713" alt="Screen Shot 2022-07-25 at 12 38 11 PM" src="https://user-images.githubusercontent.com/10121214/180860570-57da0b37-1375-47a9-966a-ba7c57d60c5d.png">